src/NuGet.Core/NuGet.Packaging/TransitivePackageReference.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.Frameworks;
+using NuGet.Packaging.Core;
+
+namespace NuGet.Packaging
+{
+    public class TransitivePackageReference : PackageReference
```

Comment:
This type is not used in any src/NuGet.Core classes and as such doesn't need to be part of the public API and can be defined in a Visual Studio specific assembly.

Reply:
Moved `TransitivePackageReference` class to `NuGet.VisualStudio.Internal.Contracts` namespace.